### ISSUE TITLE
Fixed font is overwritten by default font

### ISSUE THREAD

**Issue by pierrenicolasr** [oclero/qlementine#53]
Hello,

if you have a Widget with a fixed font (for instance a QPlainTextEdit, used to display code), the style overwrites the font with regular font:

https://github.com/oclero/qlementine/blob/0837347f44e4e4e82d9e8fa11ba49c3e2fca7b18/lib/src/style/QlementineStyle.cpp#L4784

```
  // Font.
  if (shouldHaveBoldFont(w)) {
    w->setFont(_impl->theme.fontBold);
  } else if (!qobject_cast<QLabel*>(w)) {
    w->setFont(_impl->theme.fontRegular);
  }
```

We maybe want to add shouldHaveFixedFont ?

**Comment by oclero** [comment]
Good idea 👍

**Comment by oclero** [comment]
Fixed by [ccbd34c](https://github.com/oclero/qlementine/commit/ccbd34c5d8ecfba9abce2e98eb19433fe1cf061c)
